src/plugins/intel_cpu/src/nodes/executors/acl/acl_utils.hpp

```diff
@@ -10,6 +10,28 @@
 namespace ov {
 namespace intel_cpu {
 
+/**
+* @brief ACL supports arm_compute::MAX_DIMS maximum. The method squashes the last
+* dimensions in order to comply with this limitation
+* @param dims vector of dimensions to squash
+* @return vector of dimensions that complies to ACL
+*/
+inline VectorDims reshape_sizes(VectorDims dims) {
```

Comment:
I would say this is not self-explainable name. I can propose smt like: "collapse_dims_to_max_rank".